integration-tests/suites/config/images.go

```diff
@@ -62,7 +63,16 @@ func loadImageStore(location string) (*ImageStore, error) {
 // tag as is otherwise.
 func getQATag(base_tag string) string {
 	if collectorQATag == "" {
```

Comment:
[ultra-nit] Do we need lazy loading of this variable? If we only read it once or twice, can't we just read it always and make the function slightly simpler? 

Or maybe since it's used as part of the `ImageStore` struct we could create a `func (i *ImageStore) New()` that will initialize this value internally? I could even argue the existing `loadImageStore` function could be easily refactored into `New`.

Reply:
Yeah it's read a lot over the course of a test run so it's worth caching the result, but your point about `loadImageStore` is good so I'll move it there